### PR TITLE
Lightgbm

### PR DESCRIPTION
**Pull request by teh**
###### Motivation for this change

New package + sha fix

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by disassembler** [comment]
@GrahamcOfBorg build pythonPackages.lightgbm

**Comment by FRidh** [comment]
sha was fixed in #31951?

File: pkgs/development/python-modules/lightgbm/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1f1drk3s9sfhscp3dvsxhrnn2ajhfrn65rz4gwl09lasy2b8s6wh";
+  };
+
+  propagatedBuildInputs = [ numpy scipy scikitlearn ];
+  buildInputs = [
+    cmake
+  ];
+
```
Comment (FRidh) [inline pkgs/development/python-modules/lightgbm/default.nix]:
```
Ran 0 tests in 0.000s
```
No tests are found. Either the test runner cannot find tests and needs to be patched, or there simply are no tests in which case the tests need to be disabled. Do include a comment explaining why the tests are disabled.

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/lightgbm/default.nix (hunk 1 above)
Comment (teh) [reply pkgs/development/python-modules/lightgbm/default.nix]: I disabled tests and added a comment. The tests don't seem to be included in the pypi distribution. We could build from GitHub if we want the tests?

**Comment by xeji** [comment]
@GrahamcOfBorg  build python36Packages.lightgbm python27Packages.lightgbm

**Comment by mmahut** [comment]
Are there any updates on this pull request, please?

**Comment by teh** [comment]
@mmahut I'm carrying this around in my own overlay. Might be easiest if you create a new PR if you want this included in nixpkgs itself? Happy to review that. Going to close this branch as it's way old.

**Pull request closed without merging**